### ISSUE TITLE
Development Quickstart Failing Due to SELinux

### ISSUE THREAD

**Issue by damaestro** [lyft/confidant#57]
I've followed http://lyft.github.io/confidant/advanced/contributing/#quickstart-for-testing-or-development on Fedora 23 and the application is failing to start due to SELinux:

```
type=AVC msg=audit(1459447090.791:1040): avc:  denied  { write } for  pid=12660 comm="java" name="dynamo" dev="dm-1" ino=1469994 scontext=system_u:system_r:svirt_lxc_net_t:s0:c373,c563 tcontext=system_u:object_r:mnt_t:s0 tclass=dir permissive=0
```

The application does start with SELinux permissive. Please consider using auto labeling which has been merged in docker 1.7+. http://www.projectatomic.io/blog/2015/06/using-volumes-with-docker-can-cause-problems-with-selinux/

selinux-policy-targeted-3.13.1-158.11.fc23.noarch
docker-1.9.1-6.git6ec29ef.fc23.x86_64

**Comment by damaestro** [comment]
To get things working without SELinux permissive: `sudo chcon -Rt svirt_sandbox_file_t /mnt/dynamo`

**Comment by ryan-lane** [comment]
Thanks for the info! I've fixed this via c7ca4e8206374c8c6e785b42eee732e23a8a15a5
